GCTC-NTGC/gc-digital-talent
# [Deps] Bump storybook to v7.6.1

## 👋 Introduction

Groups the storybook dependabot updates for single merge.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Install deps `npm i`
2. Run storybook `npm run storybook`
3. Confirm it works
